pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -299,6 +299,13 @@ type ClusterStatus struct {
 	CephVersion *ClusterVersion `json:"version,omitempty"`
 }
 
+// CephDaemonsVersions is a structure that can be used to parsed the output of the 'ceph versions' command
+type CephDaemonsVersions struct {
+	Mon map[string]int `json:"monVersion,omitempty"`
+	Mgr map[string]int `json:"mgrVersion,omitempty"`
+	Osd map[string]int `json:"osdVersion,omitempty"`
```

Comment:
currently, I'm only showing ceph versions of these three daemons...let me know if you want to remove/add something

Reply:
We need all of them.

Reply:
Done.